botocore/utils.py

```diff
@@ -524,8 +525,9 @@ class ArgumentGenerator(object):
         print("Sample input for dynamodb.CreateTable: %s" % sample_input)
 
     """
-    def __init__(self):
-        pass
+    def __init__(self, use_member_names=False):
+        self._use_member_names = use_member_names
```

Comment:
It's also useful if you're generating a sample output response and want to test out JMESPath queries.  Using the member name gives you enough context about what you're matching.  Similar to what I use for my re:Invent talk here: https://youtu.be/TnfqJYPjD9I?t=971